src/App.test.js

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import App from './App';
 
-test('renders App', () => {
-  render(<App />);
+const navLinkCases = [['Search'], ['How it works'], ['About']];
+
+describe('Header', () => {
+  beforeEach(() => {
+    render(
+      <App />,
+    );
+  });
+
+  it('App logo link apears on the page', () => {
+    expect(
+      screen.getByRole('link', { name: /logo/i }),
+    ).toBeInTheDocument();
+  });
+
+  test.each(navLinkCases)(
```

Comment:
I'd probably not use a const here and move the array directly here. It's a bit harder to follow what the test is doing if you have `navLinkCases` at the top of the file

Reply:
Sounds good, I will work on refactoring it.